<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>Mouse</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="checkBox_2">
     <property name="text">
      <string>Keyboard</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QListWidget" name="listWidget">
     <property name="autoScroll">
      <bool>true</bool>
     </property>
     <property name="layoutMode">
      <enum>QListView::Batched</enum>
     </property>
     <property name="currentRow">
      <number>-1</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
